pythonsdk/build.gradle

```diff
@@ -13,7 +13,7 @@ ext {
         pypi_repo = 'testpypi'
     }
     if (!project.rootProject.hasProperty('vantiqServiceSdkVersion')) {
-        vantiqServiceSdkVersion = '0.1.0'
+        vantiqServiceSdkVersion = '0.2.0'
```

Comment:
want to bump the release number. assuming minor release (ala semantic versioning) is the right number to move.